src/stores/auth.js

```diff
@@ -11,7 +12,7 @@ export const useAuthStore = defineStore({
   }),
   getters: {
     isUserLoggedIn() {
-      return !!this.token;
+      return this.token ? true : false;
```

Comment:
Why did you remove that? Also you can use `return Boolean(this.token)`

Reply:
I guess it felt more explicit. But I like your suggestion with Boolean()